CHANGELOG.md

```diff
@@ -1,25 +1,19 @@
-# NEXT RELEASE
-
-### Enhancements
-* None.
+# 5.10.3 Release notes
 
 ### Fixed
 * When a sort or distinct over links was done on an already-sorted TableView,
   the link translation map was done using the unsorted rows, resulting in the
   second sort/distinct being done with the incorrect values.
   PR [#3102](https://github.com/realm/realm-core/pull/3102).
  
-### Breaking changes
-* None.
-
 ### Compatibility
 * File format: ver. 9 (upgrades automatically from previous formats)
 
 -----------
 
 ### Internals
 
-* None.
+* Will assert if someone tries to free a null ref.
```

Comment:
I suggest adding the issue number this is attempting to diagnose.

Reply:
@bmunkholm release notes updated